app/src/main/kotlin/com/wire/android/ui/home/conversations/search/SearchPeopleViewModel.kt

```diff
@@ -208,20 +206,17 @@ abstract class SearchPeopleViewModel(
         .asStateFlow()
         .debounce(DEFAULT_SEARCH_QUERY_DEBOUNCE)
 
-    protected val initialContactResultFlow = flow {
-        when (val result = getInitialContacts()) {
-            is SearchResult.Failure -> {
-                emit(
+    suspend fun initialContactResultFlow() =
+        getInitialContacts().map { result ->
+            when (result) {
+                is SearchResult.Failure -> {
                     SearchResultState.Failure(result.failureString)
-                )
-            }
-            is SearchResult.Success -> {
-                emit(
+                }
+                is SearchResult.Success -> {
                     SearchResultState.Success(result.contacts)
-                )
+                }
             }
         }
```

Comment:
I do not think there is need for a change to a function, you can keep 

protected val initialContactResultFlow as the rest as a variable, and keep the rest you did with a map { }

Reply:
I fixed issue with using suspend fun for flowables